R/query.R

```diff
@@ -43,15 +43,23 @@
 Query <- function(queryString) {
   # Encode the query to REST URL
   urlEncodedQuery <- URLencode(queryString, reserved = TRUE)
-  reqUrl <- paste0("http://api.datacommons.org/query?sparql=", URLencode(urlEncodedQuery), "&key=", Sys.getenv("API_KEY"))
+  reqUrl <- paste0("http://api.datacommons.org/query?sparql=",
+                   URLencode(urlEncodedQuery), "&key=", Sys.getenv("API_KEY"))
   resp <- GET(reqUrl)
   if (http_type(resp) != "application/json") {
     stop("API did not return json", call. = FALSE)
   }
 
   # Parse response
-  parsedResp <- jsonlite::fromJSON(content(resp, "text"), simplifyVector = FALSE)
+  parsedResp <- jsonlite::fromJSON(content(resp, "text"),
+                                   simplifyVector = FALSE)
   if (http_error(resp)) {
+    if (str_detect(parsedResp$message, 'Method doesn\'t allow unregistered
```

Comment:
Can you check for the response error code rather than parse the error string? That seems more likely to change.